exporters/otlp/otlplog/otlploghttp/exporter.go

```diff
@@ -5,14 +5,16 @@ package otlploghttp // import "go.opentelemetry.io/otel/exporters/otlp/otlplog/o
 
 import (
 	"context"
+	"sync/atomic"
 
 	"go.opentelemetry.io/otel/sdk/log"
 )
 
 // Exporter is a OpenTelemetry log Exporter. It transports log data encoded as
 // OTLP protobufs using HTTP.
 type Exporter struct {
-	// TODO: implement.
+	client  atomic.Pointer[client]
+	stopped atomic.Bool
```

Comment:
I know this is late but... As far as I remember the exporters do not have to be concurrent safe as the processors handle synchronization. This is also why `Exporter` interface does not tell anything about concurrency as opposed to `Processor` interface.

```suggestion
	client     client
	stopped bool
```

Reply:
Can you provide a link? The only thing I am aware of is the `Export` method is not going to be called concurrently. I don't know of any restrictions on calling the different methods concurrently.

Reply:
You are right. I think we should document that Exporter methods can be called concurrently. I will try to address it in a separate PR.

Reply:
https://github.com/open-telemetry/opentelemetry-go/issues/5178